### PR TITLE
colorful piping

### PR DESCRIPTION
**Pull request by paulpflug**
fix for #7
largely copied from nodemon

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -9,14 +8,20 @@ function potentialExit (childCmd, code) {
...
+var shFlag = '-c';
 
+if (process.platform === 'win32') {
+    sh = 'cmd';
+    shFlag = '/c';
+}
 process.argv.slice(2).forEach(function (childCmd) {
-    var child = exec(childCmd, {
-        cwd: process.cwd(),
+    console.log('spawning '+childCmd);
```
Comment (keithamus) [inline index.js]: This shouldn't `console.log`, IMO.

**Comment by keithamus** [comment]
Great work @paulpflug! Get rid of the console.log and I'll merge :tada:

**Comment by paulpflug** [comment]
right, forgot it ;)

done

**Comment by mikalai-silivonik** [comment]
@keithamus so to get this change I have to wait for next release? any plans for it?:)

**Comment by keithamus** [comment]
@mikalai-silivonik released as `v1.1.0`

**Pull request merged**